.github/workflows/swift.yml

```diff
@@ -65,41 +59,24 @@ jobs:
         with:
           extra_args: clang-format --all-files
   codegen:
-    if : ${{ github.event_name == 'pull_request' }}
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: bazelbuild/setup-bazelisk@v2
```

Comment:
Do you no longer need this? Are you getting Bazel from the Actions image then?

Reply:
indeed, see https://github.com/actions/runner-images/issues/221#issuecomment-595193536. To be noted, is that on all images `bazel` actually points to a bazelisk binary, which means the correct bazel version from `.bazelversion` will be used.

Reply:
(see for example https://github.com/github/codeql/actions/runs/13896248292/job/38877429011?pr=19029#step:4:5 where bazelisk reports downloading the required version)